bundle/android/strings.xml

```diff
@@ -181,7 +181,10 @@
     <string name="gutenberg_native_featured" tools:ignore="UnusedResources">Featured</string>
     <string name="gutenberg_native_file_block_settings" tools:ignore="UnusedResources">File block settings</string>
     <string name="gutenberg_native_file_name" tools:ignore="UnusedResources">File name</string>
+    <string name="gutenberg_native_file_type_not_supported_as_a_media_file" tools:ignore="UnusedResources">File type not supported as a media file.</string>
```

Comment:
Added in https://github.com/WordPress/gutenberg/pull/56791/files#diff-f06faf4815cadf891d1ed891241c2e2bcaa2381e8a3043115d49fbdc1043d621R159